src/cmake/thirdparty/SetupHDF5.cmake

```diff
@@ -116,6 +116,8 @@ if(WIN32 AND TARGET hdf5::hdf5-shared AND BUILD_SHARED_LIBS)
     message(STATUS "HDF5 using hdf5::hdf5-shared target")
     blt_import_library(NAME      hdf5
                        LIBRARIES hdf5::hdf5-shared
+                       INCLUDES  ${HDF5_INCLUDE_DIRS}
+                       TREAT_INCLUDES_AS_SYSTEM ON
```

Comment:
@white238 -- same here for HDF5 - the include dir was not being added to the import target on msvc; this fixes the problem.

Reply:
that's strange, it shouldn't need the include dirs -- everything should be in hdf5::hdf5-shared.  That said, vcpkg could be changing things some how.

Reply:
Unfortunately I have found many targets that don't add their include directories. So this doesn't surprise me.

Reply:
Oddly, it was in `hdf5::hdf5-shared`, but somehow didn't make it through the transitive dependency in targets that depended on `hdf5`.  

Not sure what's going on exactly, since this used to work.  This might be related to logic for import libraries vs. `blt_register_library`... I'm not sure when it was last tested, but `vcpkg` was using `blt@0.3.0` previously.

Reply:
it's strange b/c conduit doens't need it:

https://github.com/LLNL/conduit/blob/80cf13780e666d343f767f4b02dba3d95f42cabb/src/cmake/thirdparty/SetupHDF5.cmake#L252

could be a downstream thing that's somehow my fault.

Reply:
Could be?  There's also the mixing of `blt_register_library` vs. `blt_import_library`, which could be a factor (?)

In any event, the proposed fix is straightforward and appears to be harmless.

Reply:
Could be?  There's also the mixing of `blt_register_library` vs. `blt_import_library`, which could be a factor (?)

In any event, the proposed fix is straightforward and appears to be harmless.